Source/WebKit/WebProcess/Inspector/ServiceWorkerDebuggableProxy.cpp

```diff
@@ -51,6 +52,7 @@ ServiceWorkerDebuggableProxy::ServiceWorkerDebuggableProxy(const String& url, We
     , m_identifier(identifier)
     , m_webProcessProxy(webProcessProxy)
 {
+    setPresentingApplicationPID(webProcessProxy.processPool().configuration().presentingApplicationPID());
```

Comment:
Should this be guarded by `#if ENABLE(REMOTE_INSPECTOR)`?

Reply:
Yes, the whole file was guarded already! 👍